metadata-ingestion/src/datahub/ingestion/run/pipeline_config.py

```diff
@@ -57,6 +57,11 @@ class FlagsConfig(ConfigModel):
         ),
     )
 
+    generate_memory_profiles: bool = Field(
```

Comment:
btw - it might be easier to make this an `Optional[Path]`

Reply:
Good idea, updated!